AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/FirebaseTestLabController.kt

```diff
@@ -134,6 +134,20 @@ internal class FirebaseTestLabController(
         }
     }
 
+    /**
+     * Enqueues a request to create a [TestMatrix] to retry failed Tests
+     * specified in the [testTargets] list for the given [testMatrix]
+     */
+    suspend fun submitRetryTests(
```

Comment:
as mentioned in the androidx cl, lets just call this `submitTests` because this component doesn't care that it is a retry. it is just a way to submit tests w/ explicit test names.